libraries/Microsoft.Bot.Builder.AI.Orchestrator/OrchestratorRecognizer.cs

```diff
@@ -168,15 +170,18 @@ public override async Task<RecognizerResult> RecognizeAsync(DialogContext dc, Sc
                 // if top scoring intent is less than threshold, return None
                 if (topScore < UnknownIntentFilterScore)
                 {
-                    ((List<Result>)recognizerResult.Properties[ResultProperty]).Insert(0, new Result() { Score = 1.0, Label = new Label() { Name = "None", Type = LabelType.Intent } });
+                    ((List<Result>)recognizerResult.Properties[ResultProperty]).Insert(0, new Result() { Score = 1.0, Label = new Label() { Name = NoneIntentLabel, Type = LabelType.Intent } });
 
                     // add all scores
                     foreach (var result in results)
                     {
-                        recognizerResult.Intents.Add(result.Label.Name, new IntentScore()
+                        if (result.Label.Name != NoneIntentLabel)
```

Comment:
None intent will be effectively removed from prediction especially when the top score is lower than the unknown threshold. Why?

Reply:
nice catch @hcyang